shared/src/main/scala/scala/collection/immutable/VectorMap.scala

```diff
@@ -0,0 +1,169 @@
+package scala.collection.immutable
```

Comment:
I would like to echo @gmethvin's concern here. I don't think we should use `package scala` here.

/cc @SethTisue